pytorch_pfn_extras/nn/modules/extended_sequential.py

```diff
@@ -39,6 +50,10 @@ def repeat(self, n_repeat: int, mode: 'str' = 'init'):
         The functions is supposed to behave the same way as `repeat`
         in `chainer`.
 
+        For user-defined module with parameters, add a ``reset_parameters`` or
```

Comment:
Could you elaborate more? e.g.,

> When the mode is set to `init` (this is the default), modules will be copied and reinitialized by calling `reset_parameters` (or `_reset_parameters`) method. To register user-defined modules with parameters or buffers to this Sequential, you need to implement `reset_parameters` method to the module to reinitialize parameters and (if necessary) buffers; otherwise the initialization cannot be performed and a warning message will be shown.

Reply:
Fixed in 00f32ab